### DISCUSSION TITLE
How to reduce width of plotly chart in dp.Report ?

### DISCUSSION THREAD

**Discussion by Bebio95** [datapane/datapane#145] [Help] [answered]
Hi, 

It seems the "width" parameter of a plotly chart is not taken into account when generating dp.Report. 

```
import plotly.express as px
import datapane as dp

df = px.data.gapminder()

fig = px.scatter(df.query("year==2007"), x="gdpPercap", y="lifeExp",size="pop", color="continent",
                 hover_name="country", log_x=True, size_max=60)

fig.update_layout(height=200,width=300)
fig.show()

Report=dp.Report(dp.Plot(fig))
Report.save('test.html',open=True)
```

The report always displays the graph on the maximum width of the page, unlike the height parameter which is taken into account.

![image](https://user-images.githubusercontent.com/80545003/130089903-ed3a7c1f-e9f7-41dd-af5d-ee8871382d5a.png)

Is there a way to handle this ? 
Thanks

**Comment by johnmicahreid** [comment, marked as answer]
Hi Bebio, you can try setting `responsive` to False, as follows: 
``` python
Report=dp.Report(dp.Plot(fig, responsive=False))
```
Let me know if that works!

**Reply by Bebio95** [reply]
Hi, 

It works perfectly :)
Thanks for your quick answer
